### PR TITLE
Fix issue with moduleName slugify filter

### PR DESCRIPTION
**Pull request by soul-wish**
Just a small fix to a moduleName filter. Now slugifying shoud work properly.
Thanks for all your great work, Sindre!

### CODE REVIEW

**Comment by sindresorhus** [comment]
What issue is this fixing?

**Comment by soul-wish** [comment]
None of them (I mean there is no open related issue in your repo). Should I create one? The problem in the following: if you try to use few words, separated by space symbol, as your module name (eg.: `test module`) on the first prompt - the result in package.json will be `test module` (which is wrong). My small fix will slugify module name and finally it will be `test-module`.

File: app/index.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ module.exports = class extends yeoman.Base {
 			name: 'moduleName',
 			message: 'What do you want to name your module?',
 			default: _s.slugify(this.appname),
-			filter: x => moduleName.slugify(x)
+			filter: x => _s.slugify(x)
```
Comment (sindresorhus) [inline app/index.js]: The correct fix would be to fix `moduleName.slugify()`. It's used there for a reason.

**Comment by sindresorhus** [comment]
> Should I create one?

No

**Comment by soul-wish** [comment]
Thanks for the code review and explanations. Seems slugifying was broken since [this PR](https://github.com/sindresorhus/generator-nm/pull/56). I've fixed scoped modules regexp (it was returning `true` for any module name on prompt, no matter scoped name or not). Now it should work just fine, so the result will be:

Input:
```
- What do you want to name your module?
> awesome module
```

Output (package.json):
```
"name": "awesome-module",
"repository": "username/awesome-module"
```

---

Input:
```
- What do you want to name your module?
> @scoped/module
```

Output (package.json):
```
"name": "@scoped/module",
"repository": "username/module"
```

**Comment by sindresorhus** [comment]
// @JonnyBurger

**Comment by sindresorhus** [comment]
Thanks :)

**Comment by JonnyBurger** [comment]
Thanks for the fix! I will fix any issues with my PR in the future, given that I have the chance to. You were too fast :) 

Cheera

**Pull request merged**